runtime/interpreter/value_accountcapabilitycontroller.go

```diff
@@ -316,6 +316,8 @@ func (v *AccountCapabilityControllerValue) ReferenceValue(
 		authorization,
 		account,
 		resultBorrowType.Type,
+		// okay to pass an empty range here because the account value is never a reference, so this can't fail
+		EmptyLocationRange,
```

Comment:
Here and below (e.g. `account.go`): Maybe pass the proper location range here.

Though that statement is true, because the location range is currently only used to report an error when the referenced value is a reference, that might not stay true in the future, the position information should be valid on its own.